astrobee/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.5)
```

Comment:
Can this be moved inside the ROS2 branch of the conditional to avoid levying the greater version requirement on ROS1? Thinking of 16.04. Or fine if it's valid for 16.04, I didn't check.

Reply:
I checked that my 16.04 cmake version was greater than 3.5